ovs-p4rt/sidecar/testing/encode_host_port_value_test.cc

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <arpa/inet.h>
+#include <stdarg.h>
+
+#include <string>
+
+#include "gtest/gtest.h"
+#include "ovsp4rt/ovs-p4rt.h"
+
+//----------------------------------------------------------------------
+// Support function (from ovsp4rt.cc)
+//----------------------------------------------------------------------
+std::string EncodeByteValue(int arg_count...) {
```

Comment:
Shouldnt we be using the actual function from the ovsp4rt.cc instead of a repeated/mock function which duplicates the code?

Reply:
It depends. Sometimes you're independently verifying something, and don't want to rely on the implementation in the function used by the Unit Under Test. I don't think that's the case here, but I'll need to double-check the code to be sure.

Good suggestion.

Reply:
On review, `EncodeByteValue()` is used to establish the control value. It should be the actual implementation, not a private copy. You're right.

I've implemented your suggestion. Good call. 👍